wordpress-mobile/WordPress-Android
# Revamp Landing Screen - Fix WordPress app brush stroke bg image getting cut

Fixes #17267

This PR provides a solution to fix an issue causing the brush stroke to get cut vertically on taller devices.

To test:

<details>
<summary><b>Prerequisite</b> - Enable the <code>LandingScreenRevamp</code> feature flag</summary>

- Either `login` → `Me` → `App Settings` → `Debug Settings` → enable `LandingScreenRevamp` → `RESTART THE APP` → `logout` or
- Apply this hard-coded solution:

<details>
<summary>Patch to enable the <code>LANDING_SCREEN_REVAMP</code> build flag</summary>

```diff
diff --git a/WordPress/build.gradle b/WordPress/build.gradle
index 63260a4ca8..fe999fa92f 100644
--- a/WordPress/build.gradle
+++ b/WordPress/build.gradle
@@ -112,7 +112,7 @@ android {
         buildConfigField "boolean", "READER_COMMENTS_MODERATION", "false"
         buildConfigField "boolean", "SITE_INTENT_QUESTION", "true"
         buildConfigField "boolean", "SITE_NAME", "false"
-        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "false"
+        buildConfigField "boolean", "LANDING_SCREEN_REVAMP", "true"
         buildConfigField "boolean", "LAND_ON_THE_EDITOR", "false"
         buildConfigField "boolean", "BLOGGING_PROMPTS", "false"
         buildConfigField "boolean", "QUICK_START_EXISTING_USERS_V2", "false"
```  
</details>

---

</details>

1. Launch the WordPress app and make sure you're logged out
2. **Expect** the brush stroke background image to not get cut (see preview images in the [issue description](https://github.com/wordpress-mobile/WordPress-Android/issues/17267))

<details>
<summary><h3>📸 Previews</h3></summary>

| Phone | Phone Dark Theme |
| - | - |
| ![pixel_6](https://user-images.githubusercontent.com/4588074/194083765-7bc3d354-7c77-4ba1-8875-ccd4e2d2fab3.png) | ![dark](https://user-images.githubusercontent.com/4588074/194083915-6c3ddd76-8dd9-4d08-9157-61f7c58a2a32.png) |

| Tablet |
| - |
|  ![tablet](https://user-images.githubusercontent.com/4588074/194083740-ffdac269-1825-4c5d-a6d6-0fbe19afc218.png) |

</details>

## Regression Notes
1. Potential unintended areas of impact
   Devices with low memory.

3. What I did to test those areas of impact (or what existing automated tests I relied on)
   Requested review from Antonis to check on a device that proved useful for such tests.

4. What automated tests I added (or what prevented me from doing so)
   N/a.

PR submission checklist:
